src/verso/Verso/Doc/TeX.lean

```diff
@@ -41,7 +41,7 @@ def logError [Monad m] (message : String) : TeXT g m Unit := do
 def header [Monad m] (name : TeX) : TeXT g m TeX := do
   let opts ← options
   let some i := opts.headerLevel
-    | logError s!"No more header nesting available at {name.asString}"; return \TeX{\textbf{\Lean{name}}}
```

Comment:
This should also be a proper error, and it should insert a notice like the other two into the document that is unmistakable.

Reply:
done.